### ISSUE TITLE
WinDbg 1.2303.30001.0 not on Store

### ISSUE THREAD

**Issue by akihikodaki** [microsoftfeedback/WinDbg-Feedback#107]
WinDbg 1.2303.30001.0 is still not on Store and says it's preview:
https://apps.microsoft.com/store/detail/windbg-preview/9PGJGD53TN86

**Comment by michael-hawker** [comment]
I just installed from the store (it's super convenient to do so) and received version 1.2306.14001.0, which doesn't have a listing on the release page?

It did prompt me to install a different version from outside the store... which was weird.

**Comment by lzybkr** [comment]
By "release page", do you mean: https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/windbg-release-notes

1.2306.14001.0 is very similar to 1.2303.30001.0 - mostly just bug fixes and no new features.

**Comment by michael-hawker** [comment]
@lzybkr yes, that page. Even if it's just bug fixes, having it listed on the release notes would be helpful to identify it as a valid release.

**Comment by xwang186** [comment]
Same request here. I would prefer to have some documentations introducing all the released version.

Also I am receiving a weird update from 14001 to 12001. I believe 14001 is currently the newest version. Am I right? @lzybkr
Then there should be some issue in the publishment.
 
![image](https://github.com/microsoftfeedback/WinDbg-Feedback/assets/39249169/e63b1d2d-4ca2-49b9-a23b-702ae2395387)
And get
![image](https://github.com/microsoftfeedback/WinDbg-Feedback/assets/39249169/8e98d887-33b9-4401-be52-7d10592b1cda)

**Comment by maolike** [comment]
I found an interesting problem, when I upgraded WinDbg from 1.12306.12001.0 to 1.2306.14001.0 in the Micorsoft Store; then I ran the windbg installer AppInstaller.exe; it prompted an update

**Comment by KenSykes** [comment]
We are cleaning up the issues list. This bug is older than six months. If you still see this issue in the current debugger and would like to see a fix please submit a new report.\ Thanks!
